Source/MovieContainer.swift

```diff
@@ -141,6 +142,19 @@ class MovieContainer: UIView {
     }
 
     func start() {
+
+        guard let player = self.player, currentItem = player.currentItem else { return }
+
+        let interval = CMTime(seconds: 0.4, preferredTimescale: Int32(NSEC_PER_SEC))
```

Comment:
It means that it updates the currentTime every 0.4 seconds. Maybe every second is better, I was trying to make it smoother. But have to find another way of doing that!